_includes/linkedin_logo.html

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="22pt" height="22pt" viewBox="0 0 22 22" version="1.1">
-<g id="surface1">
-<rect x="0" y="0" width="22" height="22" style="fill:rgb(0%,51.764706%,69.411765%);fill-opacity:1;stroke:none;"/>
-<path style=" stroke:none;fill-rule:nonzero;fill:rgb(100%,100%,100%);fill-opacity:1;" d="M 5.175781 3.339844 C 6.183594 3.34375 7.015625 4.175781 7.015625 5.1875 C 7.011719 6.246094 6.132812 7.046875 5.164062 7.023438 C 4.183594 7.042969 3.324219 6.222656 3.324219 5.179688 C 3.324219 4.167969 4.15625 3.339844 5.175781 3.339844 Z M 5.175781 3.339844 "/>
-<path style=" stroke:none;fill-rule:nonzero;fill:rgb(100%,100%,100%);fill-opacity:1;" d="M 6.25 18.65625 L 4.089844 18.65625 C 3.808594 18.65625 3.582031 18.429688 3.582031 18.148438 L 3.582031 8.9375 C 3.582031 8.660156 3.808594 8.433594 4.089844 8.433594 L 6.25 8.433594 C 6.527344 8.433594 6.753906 8.660156 6.753906 8.9375 L 6.753906 18.148438 C 6.753906 18.429688 6.527344 18.65625 6.25 18.65625 Z M 6.25 18.65625 "/>
-<path style=" stroke:none;fill-rule:nonzero;fill:rgb(100%,100%,100%);fill-opacity:1;" d="M 18.675781 11.667969 C 18.675781 9.734375 17.109375 8.171875 15.179688 8.171875 L 14.625 8.171875 C 13.570312 8.171875 12.636719 8.699219 12.074219 9.503906 C 12.015625 9.589844 11.957031 9.683594 11.90625 9.777344 C 11.886719 9.773438 11.878906 9.773438 11.878906 9.773438 L 11.878906 8.644531 C 11.878906 8.527344 11.78125 8.433594 11.667969 8.433594 L 8.972656 8.433594 C 8.855469 8.433594 8.761719 8.527344 8.761719 8.644531 L 8.761719 18.445312 C 8.761719 18.5625 8.855469 18.65625 8.972656 18.65625 L 11.726562 18.65625 C 11.839844 18.65625 11.9375 18.5625 11.9375 18.445312 L 11.9375 12.78125 C 11.9375 11.804688 12.714844 10.992188 13.695312 10.980469 C 14.195312 10.972656 14.648438 11.175781 14.976562 11.5 C 15.296875 11.824219 15.496094 12.269531 15.496094 12.761719 L 15.496094 18.449219 C 15.496094 18.5625 15.589844 18.65625 15.707031 18.65625 L 18.464844 18.660156 C 18.582031 18.660156 18.675781 18.5625 18.675781 18.449219 Z M 18.675781 11.667969 "/>
-</g>
+<svg width="22pt" height="22pt" version="1.1" viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't think width and height attributes are needed, you should be able to remove them and keep only viewBox